src/main/java/sx/blah/discord/api/internal/DiscordUtils.java

```diff
@@ -54,6 +65,9 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
+import static sx.blah.discord.handle.audit.entry.change.ChangeMap.Collector.toChangeMap;
```

Comment:
I'm not personally a fan of `import static` as it makes the code harder to read at a glance.